language/documentation/examples/diem-framework/move-packages/DPN/tests/AccountFreezingTests.move

```diff
@@ -5,7 +5,7 @@ module DiemFramework::AccountFreezingTests {
     use std::signer;
 
     #[test(tc = @TreasuryCompliance, dr = @DiemRoot)]
-    #[expected_failure(abort_code = 1)]
+    #[expected_failure(abort_code = 1, location = DiemFramework::DiemTimestamp)]
```

Comment:
I assume the downwards compat behavior is an abort code at any location matches?

Reply:
A warning is given if it is missing